icicle_v3/include/icicle/mmcs.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "errors.h"
+#include "runtime.h"
+#include "hash.h"
+#include "merkle_tree.h"
+#include "icicle/utils/utils.h"
+
+#include <cstdint>
+#include <functional>
+
+
+template <typename T>
+  struct Matrix {
+    T* values;
+    size_t width;
+    size_t height;
+  };
+
+eIcicleError build_mmcs_tree(const Matrix<limb_t>* inputs,
```

Comment:
what is this file? we don't support mmcs

Reply:
seems like a mistake. I removed it